packages/reactivity/src/computed.ts

```diff
@@ -68,10 +68,51 @@ export class ComputedRefImpl<T> {
   }
 }
 
+/**
+ * Creates an immutable reactive ref object.
```

Comment:
Let's follow the docs description for the computed as well.

Reply:
I could've sworn I've already fixed the one for `reactive()` 🤣 Anyhow, aligned with the docs page now.
For `computed()`, I took over the text from the docs and removed the second tsdoc block.